compiler/backend/presLangScript.sml

```diff
@@ -16,8 +16,14 @@ val _ = new_theory"presLang";
 (* Special operator wrapper for presLang *)
 val _ = Datatype`
   op =
-    | Ast ast$op
-    | Conlang conLang$op`;
+    | Ast_op ast$op
+    | Con_op conLang$op`;
```

Comment:
At first I didn't think of it as a problem, but going through the remaining code I can imagine the name Con_op to be a bit confusing. It could be interpretted as "The op-constructor" or something along those lines. A possible solution could be renaming it Conlang_op if we agree that it's an issue.